tipperInteractions/get_info.py

```diff
@@ -15,7 +15,7 @@ def get_info_as_string(wallet_name, private_info=False, password="\"\""):
     :return formatted string of content
     """
 
-    info = get_info(wallet_name, private_info, password)
+    info = get_info(wallet_name=wallet_name, private_info=private_info, password=password)
```

Comment:
For anyone who sees this in the future, yeah look at line 24. It's setting the port to the password, then using the default (blank) password to try and later unlock the wallets.